app/components/ListPlusCell.tsx

```diff
@@ -30,7 +30,7 @@ export const ListPlusCell = ({
   const rest = array.slice(numInCell)
   const content = (
     <div>
-      <div className="mb-2">{tooltipTitle}</div>
+      {tooltipTitle && <div className="mb-2">{tooltipTitle}</div>}
```

Comment:
Might be nice with `Other transit IPs` or something as the title? Maybe not though.

Reply:
I was ambivalent on this, but realized that I was looking at it on a larger screen:
<img width="481" alt="Screenshot 2024-09-17 at 8 43 55 AM" src="https://github.com/user-attachments/assets/6834e6d7-a29f-407f-87bd-5c147353b2bc">

When looking at it on a smaller screen, it's possible for the tooltip to cover up the column header:
<img width="281" alt="Screenshot 2024-09-17 at 8 44 25 AM" src="https://github.com/user-attachments/assets/85a7427d-bef1-4a0c-bb64-9ff13dec7141">

So I think having the header in makes it a bit clearer:
<img width="270" alt="Screenshot 2024-09-17 at 8 43 18 AM" src="https://github.com/user-attachments/assets/8aee0174-d77b-4745-aa0a-e68a00b31f5f">

Will add that in.